### PR TITLE
cli-progress: add missing property to `Params`

### PR DESCRIPTION
**Pull request by in-pursuit-of-ash**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/npkgz/cli-progress/blob/31692f2610e58fa40069d69ec3da329d431076d3/lib/generic-bar.js#L60
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

What changed:
- A property was added to `Props` that is in `cli-progress` v3.11 (and possibly previous versions), but was not in the type declarations

### CODE REVIEW

File: types/cli-progress/index.d.ts [hunk 1]
```diff
@@ -7,12 +7,13 @@
...
 /// <reference types="node" />
 
-import EventEmitter = require("events");
+import EventEmitter = require('events');
 
 export interface Params {
     progress: number;
     eta: number;
     startTime: number;
+    stopTime: number | null;
```
Comment (peterblazejewicz) [inline types/cli-progress/index.d.ts:16]:
```suggestion
    stopTime: number;
```
as with `startTime`, the default options are skipped in the interface (intentionally IMO)
https://github.com/npkgz/cli-progress/blob/e2347a35a6c0922cfb0a077cf5ed21696fba46da/lib/generic-bar.js#L32

Comment (frantisekhanzlikbl) [reply types/cli-progress/index.d.ts:16]: I believe with `startTime`, the idea was that if the progress isn't "started" (i.e. `startTime = null`), the formatting function won't run and therefore the null can't really be observed. that wouldn't be true about the `stopTime` though, as the formatting function runs both before and after the progress stops.

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@frantisekhanzlikbl thanks!

**Comment by frantisekhanzlikbl** [comment]
Ready to merge

**Pull request merged**